utils/read_file.py

```diff
@@ -0,0 +1,20 @@
+import pandas as pd
+
+def read_file(filename, **kwargs):
+
+    """Read file with **kwargs; files supported: xls, xlsx, csv, csv.gz, pkl"""
+
+    read_map = {
+    	'xls': pd.read_excel,
+    	'xlsx': pd.read_excel,
+    	'csv': pd.read_csv,
+    	'gz': pd.read_csv,
+    	'pkl': pd.read_pickle
+    }
+
+    file_ext = filename.name.split('.')[1].lower()
```

Comment:
A very bad way to check for the file type. One should always check a file's mime-type to determine what type is it. https://docs.python.org/3.8/library/mimetypes.html